js/checkbox-wikirule.js

```diff
@@ -21,30 +21,23 @@ exports.init = function(parser) {
 };
 
 /*
-Retrieve the configuration state indicating if items should be striken
+Retrieve the configuration state of the clear all button
 */
-exports.shouldStrikeChecked = function() {
```

Comment:
We can check this directly from the CSS now, see below. As stirkethrough is a styling issue, I thought this would be the right place.